src/thermopro_ble/parser.py

```diff
@@ -63,6 +71,46 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
         if len(data) < 6:
             return
 
+        if name.startswith("TP96"):
+            MAX_BAT = 2880
+            MIN_BAT = 1600  # ??
```

Comment:
Please make constants above and make it clear the are for the TP96 series in the name

Reply:
Makes sense, will do